components/datadog/apps/nginx/ecsFargate.go

```diff
@@ -91,14 +90,14 @@ func FargateAppDefinition(e aws.Environment, clusterArn pulumi.StringInput, apiK
 		LogConfiguration: ecsClient.GetFirelensLogConfiguration(pulumi.String("nginx"), pulumi.String("nginx"), apiKeySSMParamName),
 	}
 
-	serverTaskDef, err := ecsClient.FargateTaskDefinitionWithAgent(e, e.CommonNamer.ResourceName("nginx-server"), e.CommonNamer.DisplayName(255, pulumi.String("server")), 1024, 2048, map[string]ecs.TaskDefinitionContainerDefinitionArgs{"server": *serverContainer}, apiKeySSMParamName, fakeintake, opts...)
+	serverTaskDef, err := ecsClient.FargateTaskDefinitionWithAgent(e, "nginx", pulumi.String("nginx-fg"), 1024, 2048, map[string]ecs.TaskDefinitionContainerDefinitionArgs{"nginx": *serverContainer}, apiKeySSMParamName, fakeintake, opts...)
 	if err != nil {
 		return nil, err
 	}
 
 	if _, err := ecs.NewFargateService(e.Ctx, namer.ResourceName("server"), &ecs.FargateServiceArgs{
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Is it meant to leave this as `server` ?

Reply:
The `namer` object already contains the `nginx` prefix.
Then, this namer is used for a `server` fargate service and for a `query` one.